<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>SOUL</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>801</width>
         <height>551</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
        <item>
         <widget class="QWebEngineView" name="MapWidget" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>接收框：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_RcvMsg">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_SendMsg">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>20</y>
         <width>261</width>
         <height>351</height>
        </rect>
       </property>
       <property name="title">
        <string>串口参数</string>
       </property>
       <widget class="QCheckBox" name="openCom">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>231</width>
          <height>261</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>串口：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="portName"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>数据位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="dataBits">
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>波特率：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="baudRate">
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38400</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>停止位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="stopBits">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>校验位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="parity">
             <item>
              <property name="text">
               <string>NoParity</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>EvenParity</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>OddParity</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>370</y>
         <width>261</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>GPS信息</string>
       </property>
       <widget class="QTextBrowser" name="comLog">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>241</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="message"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
